tests/conftest.py

```diff
@@ -5,7 +5,6 @@
 
 import pyarrow
 import pytest
-from _pytest.logging import caplog as _caplog
```

Comment:
Thank you for fixing this. It's quite annoying to have it being removed every time it's formatted and then break the test.